source/fundamentals/connection/connection-options.txt

```diff
@@ -141,20 +191,26 @@ parameters of the connection URI to specify the behavior of the client.
        is greater. For more information, see the server documentation for the
        :manual:`maxStalenessSeconds option </reference/connection-string/#urioption.maxStalenessSeconds>`.
 
+       | **Default**: ``-1`` (no staleness check)
```

Comment:
issue: In the other settings where an explanation of a special value was helpful you added it there. I suggest adding it here as well, and then removing the parenthetical.

```suggestion
       | **Default**: ``-1``
```

Reply:
Removed note next to `-1` value. There was already an explanation for this value present in the description, but I can see how now it's inconsistent with other parameters that list special values at the end.

Moved description of `-1` to the end of the description.